README.md

```diff
@@ -52,13 +52,19 @@ This is a work-in-progress implementation of a service broker for services provi
 `PASSWORD` - username password
 `DB_PREFIX` - a prefix that can be used to tag instances. Defaults to `compose-broker`
 `CLUSTER_NAME` - a name of your enterprise cluster if you've got one and want to use it. Defaults to hosted compose
-`ACCESS_TOKEN` - your API key for Compose.
+`COMPOSE_ACCESS_TOKEN` - your API key for Compose.
 
-## Running tests locally
+## Running local tests
 
-The integration tests communicate with MongoDB server(s). You will need to run a MongoDB server on `localhost` before running local (non-Compose) tests.
+You **must** have at least version 1.4.0 of ginkgo installed
```

Comment:
Ok, It would be good to leave a note somewhere about the reason. Commit message?